lib/al/Library/Layout/LayoutActorUtil.h

```diff
@@ -1,19 +1,109 @@
 #pragma once
 
+#include <common/aglTextureData.h>
+#include <gfx/seadColor.h>
+#include <math/seadMatrix.h>
 #include <math/seadVector.h>
 
+namespace nn::ui2d {
+class TextureInfo;
+}
+
 namespace al {
 class LayoutActor;
 class IUseLayout;
+class MessageHolder;
 
-bool isActive(al::LayoutActor* actor);
-bool isDead(al::LayoutActor* actor);
-
-void killLayoutIfActive(al::LayoutActor* actor);
-void appearLayoutIfDead(al::LayoutActor* actor);
-
-void calcTrans(sead::Vector3f* trans, const al::IUseLayout*);
-const sead::Vector3f& getLocalTrans(const al::IUseLayout*);
-
-void updateLayoutPaneRecursive(al::LayoutActor* actor);
+void killLayoutIfActive(al::LayoutActor*);
+void appearLayoutIfDead(al::LayoutActor*);
+bool isActive(al::LayoutActor const*);
+bool isDead(al::LayoutActor const*);
+void calcTrans(sead::Vector3f*, al::IUseLayout const*);
+void getLocalTrans(al::IUseLayout const*);
+void getLocalTransPtr(al::IUseLayout const*);
+void calcScale(sead::Vector3f*, al::IUseLayout const*);
+void getLocalScale(al::IUseLayout const*);
+void setLocalTrans(al::IUseLayout*, sead::Vector3f const&);
+void setLocalTrans(al::IUseLayout*, sead::Vector2<float> const&);
+void setLocalScale(al::IUseLayout*, float);
+void setLocalScale(al::IUseLayout*, sead::Vector2<float> const&);
+void setLocalAlpha(al::IUseLayout*, float);
+void calcPaneTrans(sead::Vector3f*, al::IUseLayout const*, char const*);
+void calcPaneMtx(sead::Matrix34f*, al::IUseLayout const*, char const*);
+void calcPaneTrans(sead::Vector2<float>*, al::IUseLayout const*, char const*);
+void calcPaneScale(sead::Vector3f*, al::IUseLayout const*, char const*);
+void calcPaneSize(sead::Vector3f*, al::IUseLayout const*, char const*);
+void getPaneMtx(al::IUseLayout const*, char const*);
+void getPaneMtxRaw(al::IUseLayout const*, char const*);
+void getGlobalAlpha(al::IUseLayout const*, char const*);
+void setPaneLocalTrans(al::IUseLayout*, char const*, sead::Vector2<float> const&);
+void setPaneLocalTrans(al::IUseLayout*, char const*, sead::Vector3f const&);
+void setPaneLocalRotate(al::IUseLayout*, char const*, sead::Vector3f const&);
+void setPaneLocalScale(al::IUseLayout*, char const*, sead::Vector2<float> const&);
```

Comment:
```suggestion
void setPaneLocalScale(al::IUseLayout*, char const*, sead::Vector2f const&);
```
Same for other occurences in file